monitoring/README.md

```diff
@@ -338,21 +345,81 @@ For large numbers, use the `short` `Unit`.
 
 The `Color Scheme` should remain set to `Classic Palette` to help standardize our visual experience, but sometimes `Green -> Red` or `Red -> Green` palettes are ideal.
 
+#### Data Links
+
+[Data Links](https://grafana.com/docs/grafana/next/panels/configure-data-links/#data-links) allow any point within a Time Series Panel to contain embedded links when clicking on a point. This can be useful for shortcuts to external websites, among other unexplored use cases.
+
+In future iterations, Data Links will be extracted and included in our auto-generated Alerts once we activate AlertManager in future iterations to enable rich Slack alerts.
+
```

Comment:
Example screenshot of data links in the panel UI:

![Screen Shot 2022-08-23 at 11 42 20 AM](https://user-images.githubusercontent.com/545666/186214539-3ef598aa-080a-4b59-ba85-161b25e75252.png)